zerver/actions/message_edit.py

```diff
@@ -990,8 +1002,10 @@ def check_update_message(
         assert message.is_stream_message()
         if not user_profile.can_move_messages_between_streams():
             raise JsonableError(_("You don't have permission to move this message"))
+        assert stream is not None
         try:
-            access_stream_by_id(user_profile, message.recipient.type_id)
+            error = _("Invalid stream")
+            access_stream_common(user_profile, stream, error)
```

Comment:
This error is actually not used as the error is handled below in `except` block, but error is a required parameter.